src/main/drivers/pwm_output.c

```diff
@@ -461,7 +464,7 @@ void servoDevInit(const servoDevConfig_t *servoConfig)
             break;
         }
 
-        pwmOutConfig(&servos[servoIndex], timer, PWM_TIMER_MHZ, 1000000 / servoConfig->servoPwmRate, servoConfig->servoCenterPulse, 0);
+        pwmOutConfig(&servos[servoIndex], timer, PWM_TIMER_HZ, 1e6 / servoConfig->servoPwmRate, servoConfig->servoCenterPulse, 0);
```

Comment:
1e6 is floating point value - servoPwmRate is converted to float, float division is performed and that resulting value is converted to int again ... It will consume considerable amount of flash and will be slow on without FPU.

Also this should be `PWM_TIMER_HZ / servoConfig->servoPwmRate`. And `servoCenterPulse` needs to compensate for PWM_TIMER_HZ value ...